models/marts/ledger_current_state/claimable_balances_current.sql

```diff
@@ -0,0 +1,66 @@
+{{
+    config(
+        tags = ["current_state"],
+        materialized = "incremental",
+        unique_key = "balance_id"
+    )
+}}
+/* Returns the latest state of each claimable balance in the `claimable_balances` table.
+
+    Rank all rows for a claimable balance by closed_at timestamp and pick the latest one.*/
+
+with
+    current_balance as (
+        select
+            cb.balance_id
+            , cb.claimants
+            , cb.asset_type
+            , cb.asset_code
+            , cb.asset_issuer
+            , cb.asset_amount
+            , cb.sponsor
+            , cb.flags
+            , cb.last_modified_ledger
+            , cb.ledger_entry_change
+            , cb.deleted
+            , cb.batch_id
+            , cb.batch_run_date
+            , cb.batch_insert_ts
+            , cb.closed_at
+            , cb.ledger_sequence
+            , row_number()
+                over (
+                    partition by cb.balance_id
+                    order by cb.closed_at desc
+                ) as rn
+        from {{ ref('stg_claimable_balances') }} as cb
+        {% if is_incremental() %}
+            -- limit the number of partitions fetched incrementally
+            where
+                cb.closed_at >= timestamp_sub(current_timestamp(), interval 30 day)
```

Comment:
Is `claimable_balances` partitioned on `closed_at`? We should be limiting the data selected by partition key, so I would double check the table schema.

Reply:
I think `stg_claimable_balances` is a view so it won't need it's own partition definition. It should use the same partitioning from `claimable_balances` (which is batch_run_date unfortunately)

Reply:
Addressed in https://github.com/stellar/stellar-dbt-public/pull/66/commits/fad7bb31de3760302cfbb9cd64c2b077e71a8832